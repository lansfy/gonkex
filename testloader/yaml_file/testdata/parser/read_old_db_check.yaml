- name: loader MUST read tests with obsolete database check
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  dbQuery: "SELECT ALL"
  dbResponse:
    - item1
    - item2
