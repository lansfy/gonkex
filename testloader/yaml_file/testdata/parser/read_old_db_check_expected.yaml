- GetName: loader MUST read tests with obsolete database check
  GetMethod: GET
  Path: /some/path
  GetResponses:
    200: "aaabbb"
  GetDatabaseChecks:
    - DbQueryString: "SELECT ALL"
      DbResponseJson:
        - item1
        - item2
  GetFileName: testdata/parser/read_old_db_check.yaml
  FirstTestInFile: true
  LastTestInFile: true
